bgzf.c

```diff
@@ -531,12 +531,18 @@ static int bgzf_uncompress(uint8_t *dst, size_t *dlen, const uint8_t *src, size_
         return -1;
     }
 
+    uint32_t c1 = libdeflate_crc32(0L, (unsigned char *)dst, *dlen);
```

Comment:
A minor note: `libdeflate_crc32 ` first argument is a 32 bit `int`, so a simple `0` would do.